test/actions/katello/organization_test.rb

```diff
@@ -144,6 +144,8 @@ class ManifestRefreshTest < TestBase
       rhel7 = katello_repositories(:rhel_7_x86_64)
       acme_org.stubs(:owner_details).returns({})
       acme_org.products.stubs(:redhat).returns([rhel7.product])
+      acme_org.stubs(:simple_content_access).returns(false)
+      Organization.any_instance.stubs(:simple_content_access?).returns(false)
```

Comment:
Is this stub necessary? Seems like the one directly above should be sufficient

Reply:
Fixed and removed.